packages/ui5-application-writer/src/data/defaults.ts

```diff
@@ -30,21 +31,21 @@ export function packageDefaults(version?: string, description?: string): Partial
  * @param {App} app - specifies the application properties
  * @returns {Partial<App>} the App instance
  */
-export function appDefaults(app: App): App {
-    return {
+export function mergeApp(app: App): App {
+    // Return merged, does not update passed ref
```

Comment:
Would `mergeWithDefaults` be a more descriptive name?

Probably makes sense to move the comment about not changing the reference to be part of the doc-comment? That's part of the contract, isn't it?

Reply:
I wouldn't change it. We have the `mergeWithDefaults` that is called that is calling `mergeApp` and `mergeUi5` which seems to me good enough BUT if you insist, I am ok changing it.

Reply:
It's ok for now I guess. Thanks.